vscode/src/non-stop/FixupController.ts

```diff
@@ -246,10 +246,11 @@ export class FixupController
         const nonEmptyStartIndex = editor.document.lineAt(range.start.line).firstNonWhitespaceCharacterIndex
         // add indentation to each line
         const textLines = text.split('\n').map(line => ' '.repeat(nonEmptyStartIndex) + line)
-
+        // join text with new lines, and then remove everything after the last new line if it only contains white spaces
+        const replacementText = textLines.join('\n').replace(/ +$/, '')
```

Comment:
Looks good, you could use `/[ \t]+$/` to handle tabs here too, for when we match tabs indentation.
```suggestion
        const replacementText = textLines.join('\n').replace(/[ \t]+$/, '')
```